resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -3,6 +3,9 @@
         "default_action": "trap",
         "filter_action": "allow",
         "filter": [
+            {
+                "syscall": "newfstatat"
```

Comment:
I'll add the comments and mention it in the commit message. I prefer to have the change in 47cfc024ee76adf6e41047069375a47273fb3c23 though because without it, that commit doesnt work

Reply:
An obvious use case is when we perform a task related to seccomp, eg we assess how it affects performance or whether syscalls are in use, the commit will be having a clear description and easily (temporarily) revertable if needed. What is the gain from not doing that? :)